libs/remix-simulator/src/VmProxy.ts

```diff
@@ -278,7 +278,8 @@ export class VmProxy {
         }
         let consoleArgs = iface.decodeFunctionData(functionDesc, payload)
         consoleArgs = consoleArgs.map((value) => {
-          if (isBigNumber(value)) {
+          // Copied from: https://github.com/web3/web3.js/blob/e68194bdc590d811d4bf66dde12f99659861a110/packages/web3-utils/src/utils.js#L48C10-L48C10
+          if (value && value.constructor && value.constructor.name === 'BigNumber') {
```

Comment:
This should also be considered when making a decision on what to do about the use of `BN` in `1.x`